vignettes/progeny.Rmd

```diff
@@ -157,9 +158,21 @@ this without worrying about the order of genes in the expression matrix using:
 
 ```{r}
 library(progeny)
-pathways = progeny(gene_expr)
+pathways <- progeny(gene_expr)
```

Comment:
Maybe be more explicit here with the function arguments

Reply:
I did not want to tout this much, since it is the original Mike's vignette

Reply:
I would maybe set the parameters by default in a explicity way. We show them, but the vignette keeps working as before.